django_project/lesson/models/further_reading.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+"""Further reading model definitions for lesson apps.
+
+"""
+import logging
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from lesson.models.worksheet import Worksheet
+
+logger = logging.getLogger(__name__)
+
+
+class FurtherReading(models.Model):
+    """Further reading lesson model."""
+
+    worksheet = models.ForeignKey(Worksheet)
+
+    text = models.CharField(
+        help_text=_('Text of the further reading.'),
```

Comment:
Further reading text

Reply:
This should be null false blank false.

Reply:
Both are `False` by default according to this https://docs.djangoproject.com/en/2.0/ref/models/fields/#null

Should I make it explicit? Actually I was writing `null=False, blank=False` everywhere, and delete them since it's already the default value.

Reply:
Ah I just read your next comment, I will make it explicit.

Reply:
Thanks